benches/transform.rs

```diff
@@ -72,54 +74,78 @@ pub fn av1_iadst8(c: &mut Criterion) {
   });
 }
 
-pub fn daala_fdct4(c: &mut Criterion) {
-  let (coeffs, _) = init_buffers(4);
-
-  c.bench_function("daala_fdct4", move |b| {
-    b.iter_batched(
-      || coeffs.clone(),
-      |mut coeffs| transform::forward::native::daala_fdct4(&mut coeffs[..]),
-      BatchSize::SmallInput,
-    )
-  });
+fn get_valid_txfm_types(tx_size: TxSize) -> Vec<TxType> {
```

Comment:
it is for benchmarks so we don't care much, you can return references to 'static arrays in case you need that in other code.